examples/simple-mcp/client.py

```
@@ -0,0 +1,35 @@
+import asyncio
+from mcp.types import AnyUrl
+from mcp.client.session import ClientSession
+from mcp.client.stdio import StdioServerParameters, stdio_client
+
+
+async def main():
+    async with stdio_client(
+        StdioServerParameters(command="python", args=["weather.py"])
+    ) as (read, write):
+        async with ClientSession(read, write) as session:
```

Ruff (0.8.2)

Use a single `with` statement with multiple contexts instead of nested `with` statements

Combine `with` statements

(SIM117)